aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/services/s3/UnauthenticatedIntegrationTest.java

```diff
@@ -65,7 +65,6 @@ public void tearDown() {
     @Test
     public void testUnauthenticatedOperations() {
         s3.createBucket(bucketName);
-        s3.setBucketAcl(bucketName, CannedAccessControlList.PublicRead);
```

Comment:
Is this change sufficient? How will the below line that calls ` PutObjectRequest` succeed? Did you run these tests?

Reply:
So essentially this line was unnecessary and the test still passes without it?

Reply:
Yeah these seem redundant and the tests still pass without it, so removing would suffice in this case.